src/components/implementation/no_interface/scheddev/sl.c

```diff
@@ -104,11 +104,10 @@ sl_thd_wakeup(thdid_t tid)
 }
 
 void
-sl_thd_yield(thdid_t tid)
+sl_thd_yield_cs_exit(thdid_t tid)
```

Comment:
Why do we need this API? 
I'm trying to understand what's the motivation for letting user of this API handle sched locking, and expect only to do unlocking here.